docs/shortfin/llm/user/e2e_llama8b_k8s.md

```diff
@@ -0,0 +1,42 @@
+# LLama 8b GPU instructions on Kubernetes
```

Comment:
nit: `Llama`, not `LLama` (I think the caps changed at some point, all I see now is `Llama`)

```suggestion
# Llama 8b GPU instructions on Kubernetes
```

Reply:
weird github lol